transports/msmq/index.md

```diff
@@ -71,18 +71,34 @@ Although MSMQ has the concept of both [Public and Private queues](https://techne
 
 ## Permissions
 
-| Group          | Permissions          |
-|----------------|----------------------|
-| Owning account | Write, Receive, Peek |
-| Administrators | Full                 |
-| Anonymous      | Write                |
-| Everyone       | Write                |
+| Group          | Permissions         | Description                       |
+|----------------|---------------------|-----------------------------------|
+| Owning account | Send, Receive, Peek | Set by NServiceBus                |
+| Administrators | Full                | Set by NServiceBus                |
+| Anonymous      | Send                | Set by NServiceBus prior to 6.1.0 |
+| Everyone       | Send                | Set by NServiceBus prior to 6.1.0 |
+
+
+For endpoints using the publish-subscribe pattern, both the publisher and the subscriber must have permission to send messages to each other. This is to allow the Subscribe message to be sent from the subscribing endpoint to the publishing endpoint.
```

Comment:
I thought it was less obvious for Pub-Sub which is why I thought it needed an explicit mention. You are right though, for people who are coming from a synchronous background (like HTTP or web services) it may not be obvious. Good catch.